tests/PSModuleTest.Tests.ps1

```diff
@@ -35,14 +41,71 @@
                 $result | Should -Be '1000 microseconds'
             }
             # Test microsecond abbreviation
             It 'Format-TimeSpan - Forces formatting in microseconds with abbreviation' {
-                $result = [TimeSpan]::FromMilliseconds(1) | Format-TimeSpan -BaseUnit 'Microseconds'
+                $result = [TimeSpan]::FromMilliseconds(1) | Format-TimeSpan -BaseUnit 'Microseconds' -Format Abbreviation
+                $result | Should -Be '1000usec'
+            }
+        }
+
+        Context 'Format-TimeSpan - Format Parameter' {
+            It 'Format-TimeSpan - Formats 90 minutes using symbols (default)' {
+                $result = New-TimeSpan -Minutes 90 | Format-TimeSpan -Format Symbol
+                $result | Should -Be '2h'
+            }
+            
+            It 'Format-TimeSpan - Formats 90 minutes using abbreviations' {
+                $result = New-TimeSpan -Minutes 90 | Format-TimeSpan -Format Abbreviation
+                $result | Should -Be '2hr'
+            }
+            
+            It 'Format-TimeSpan - Formats 5 minutes using symbols' {
+                $result = New-TimeSpan -Minutes 5 | Format-TimeSpan -Format Symbol
+                $result | Should -Be '5m'
+            }
+            
+            It 'Format-TimeSpan - Formats seconds using symbols' {
+                $result = [TimeSpan]::FromMinutes(2) | Format-TimeSpan -BaseUnit 'Seconds' -Format Symbol
+                $result | Should -Be '120s'
+            }
+            
+            It 'Format-TimeSpan - Formats microseconds using symbols' {
+                $result = [TimeSpan]::FromMilliseconds(1) | Format-TimeSpan -BaseUnit 'Microseconds' -Format Symbol
                 $result | Should -Be '1000µs'
             }
+            
+            It 'Format-TimeSpan - Formats negative 5 minutes using symbols' {
+                $result = New-TimeSpan -Minutes -5 | Format-TimeSpan -Format Symbol
+                $result | Should -Be '-5m'
+            }
+
+            It 'Format-TimeSpan - Formats days using symbols' {
+                $result = New-TimeSpan -Days 3 | Format-TimeSpan -Format Symbol
+                $result | Should -Be '3d'
+            }
+
+            It 'Format-TimeSpan - Formats weeks using symbols' {
+                $result = New-TimeSpan -Days 14 | Format-TimeSpan -Format Symbol
+                $result | Should -Be '2wk'
+            }
+
+            It 'Format-TimeSpan - Formats milliseconds using symbols' {
+                $result = [TimeSpan]::FromMilliseconds(500) | Format-TimeSpan -Format Symbol
+                $result | Should -Be '500ms'
+            }
+
+            It 'Format-TimeSpan - Formats using FullNameWithAlternative format' {
+                $result = New-TimeSpan -Minutes 5 | Format-TimeSpan -Format FullNameWithAlternative
+                $result | Should -Be '5 minutes/minute'
```

Comment:
Think you misunderstood me here. "minutes/minute" and "seconds/second" was mean to say that it should choose one or the other based on it being 1 or multiple of that unit.